pkg/apis/core/types.go

```diff
@@ -3517,15 +3533,16 @@ type ServiceSpec struct {
 	// +optional
 	PublishNotReadyAddresses bool
 
-	// ipFamily specifies whether this Service has a preference for a particular IP family (e.g. IPv4 vs.
-	// IPv6).  If a specific IP family is requested, the clusterIP field will be allocated from that family, if it is
+	// ipFamilies specifies whether this Service has a preference for a particular IP family (e.g. IPv4 vs.
+	// IPv6). And specififes weather the IPs allocated to this Service must be in certain order.If a specific
+	// IP families are requested, the clusterIP/ClusterIPs fields will be allocated from these families, if they are
 	// available in the cluster.  If no IP family is requested, the cluster's primary IP family will be used.
 	// Other IP fields (loadBalancerIP, loadBalancerSourceRanges, externalIPs) and controllers which
 	// allocate external load-balancers should use the same IP family.  Endpoints for this Service will be of
 	// this family.  This field is immutable after creation. Assigning a ServiceIPFamily not available in the
 	// cluster (e.g. IPv6 in IPv4 only cluster) is an error condition and will fail during clusterIP assignment.
 	// +optional
-	IPFamily *IPFamily
+	IPFamilies []IPFamily
```

Comment:
I don't know if we can drop directly `IPFamily *IPFamily` since it was in alpha :thinking: 
I think that in EndpointSlices it kept the deprecated field for some cycles